src/app/ReactQueryProvider/constants.ts

```diff
@@ -4,6 +4,7 @@ import { API_BASE_URL } from "@/shared/constants";
 export const ERROR_MESSAGE = {
   ACCESS_TOKEN_INVALIDATED: "토큰 검증에 실패했습니다.",
   REFRESH_TOKEN_INVALIDATED: "RefreshToken 검증에 실패했습니다.",
+  INTERNAL_SERVER_ERROR: "문제가 발생했습니다.잠시 후 다시 이용해 주세요",
```

Comment:
`.` 다음에 띄어쓰기 해야할 것 같아요!

```suggestion
  INTERNAL_SERVER_ERROR: "문제가 발생했습니다. 잠시 후 다시 이용해 주세요.",
```